lib/msal-node/src/cache/NodeStorage.ts

```diff
@@ -23,8 +23,9 @@ import { InMemoryCache, JsonCache, CacheKVStore } from "./serializer/SerializerT
 
 /**
  * This class implements Storage for node, reading cache from user specified storage location or an  extension library
+ * @public
  */
-export class Storage extends CacheManager {
+export class NodeStorage extends CacheManager {
```

Comment:
Had to rename this as `Storage` is conflicting with some global symbol while generating the `index.d.ts` and auto generating the name as `Storage_2` in the final API report. There is an open issue in github regarding this: https://github.com/microsoft/rushstack/issues/1830 

Henceforth had to rename. It also is consistent with our general naming convention for Browser too.

Reply:
Is this why the other common exports needed to be renamed?

Reply:
Similar reasons, added a different note for the reason below.